build/gen_stub.php

```diff
@@ -1864,6 +1868,28 @@ private function getParameterSection(DOMDocument $doc): DOMElement {
         return $parametersRefSec;
     }
 
+    /** @param array<string, FuncInfo> $aliasMap */
+    public function createDeprecationWarning(DOMDocument $doc, array $aliasMap): DOMElement {
+        /* Creation of <refsynopsisdiv> */
+        $refsynopsisDiv = $doc->createElement('refsynopsisdiv');
+        $deprecationEntity = $doc->createEntityReference(
+            'warn.deprecated.' . (isset($aliasMap[$this->name->__toString()]) ? 'alias' : 'function-') .
+            str_replace(".", "-", $this->deprecatedSince) . "-0"
```

Comment:
@TimWolla The `since` property currently doesn't contain the patch version. However, theoretically it's possible to deprecate symbols in patch versions as well... So I'm wondering whether the `.0` suffix could be added to the property value?

Reply:
Also, currently, 3rd party extensions shouldn't really use `Deprecated` attributes... because currently I have to assume that the value of the `since` property relates to PHP versions, and not the version of the extension itself.

Reply:
> However, theoretically it's possible to deprecate symbols in patch versions as well... So I'm wondering whether the .0 suffix could be added to the property value?

I don't see a value-add in that. Deprecations in patch version should be very rare. As an example, if folks convert deprecations to Exceptions this would cause a break. Yes, folks should not do that, but they do.

Pretending that the deprecation was there in the `.0` version is a good enough approximation, because only the latest patch version of a branch is supported anyways.

Reply:
> Also, currently, 3rd party extensions shouldn't really use `Deprecated` attributes... because currently I have to assume that the value of the `since` property relates to PHP versions, and not the version of the extension itself.

see: https://externals.io/message/123397#124367